src/store/bridge.ts

```diff
@@ -71,16 +68,26 @@ async function waitForRequest (hash: string): Promise<BridgeRequest> {
   switch (request.status) {
     case BridgeTxStatus.Failed:
     case BridgeTxStatus.Frozen:
-      throw new Error('Transaction was failed or canceled')
+      throw new Error(txFailedMessage)
     case BridgeTxStatus.Done:
       return request
   }
   return await waitForRequest(hash)
 }
 
+async function waitForEthereumHash (id: string | undefined) {
```

Comment:
why do you need it?

Reply:
To wait the result of transaction if it was signed already.